### ISSUE TITLE
Nested grid concentrations much higher than coarse grid simulation?

### ISSUE THREAD

**Issue by tessac2** [geoschem/geos-chem#2030]
### Name and Institution (Required)

Name: Tessa Clarizio
Institution: UIUC

### Confirm you have reviewed the following documentation

- [x ] [Support guidelines](https://geos-chem.readthedocs.io/en/stable/help-and-reference/SUPPORT.html)
- [ x] [User manuals](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/related-docs.html)
- [ x] [Debugging GEOS-Chem and HEMCO errors](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/debug-guide.html)
- [ x] [Current and past Github issues](https://github.com/geoschem/geos-chem/issues)

### Description of your issue or question
When I run the nested grid simulation (v13.2.1, 72 vertical levels, GEOS-FP meteorology) over North America at 0.25x0.3125, I keep getting very high PM2.5 concentrations. These concentrations are much higher than the 4x5 grid which the boundary conditions are from. I have tried for a couple different years (2020 with 2019 spin up; 2021 with 2020 spin up) and get similar results. For instance, I have included some figures from August 2021 of average monthly PM2.5 concentrations at the surface level from the nested grid simulation, the nested grid regridded to 4x5 and the original 4x5 simulation. As shown, the max concentration goes up >300 ug/m3 in the nested simulation but in the 4x5 simulation it is much lower (closer to 50 ug/m3). 

The main modifications I have is that I changed the mixing_mod to 0 out any negative concentrations ( #1635). I am using the met field GEOS-FP and am using QFED emissions (following directions from #579). 

I am not sure if the issue I am having is related to Issue #824 and  the change in #1895. My HEMCO_Config.rc has the EFY extension for the GEOS-Chem boundary condition file which was one of the recommendations for the previous issues. 

Please let me know if this is a common issue or if you have any insight as to why this may be happening and how to fix! Thank you! 
![image](https://github.com/geoschem/geos-chem/assets/70339432/37fcd559-4102-49cd-b249-1c255b7226e4)
![image](https://github.com/geoschem/geos-chem/assets/70339432/c33e24a9-afa1-45f9-afeb-829e745b61fb)
![image](https://github.com/geoschem/geos-chem/assets/70339432/e4fea8b7-2ff9-4b61-afea-bdb0b3ffa744)

**Comment by yantosca** [comment]
Hi @tessac2, thanks for writing and thanks for your patience.  (I was away last week.)  

I would probably expect higher concentrations at the finer resolutions, because at the coarser resolution, the effects of emission, chemistry, etc. are diluted over a larger grid box than at nested reslolution.

Also, are you using offline emissions (biomass, dust, biogenic, seasalt)?  If you are using the online HEMCO extensions (i.e. GFED, DustDead, Megan, etc.) then those emissions will probably be larger due to the fact that they depend strongly on meterological parameters such as 10-m winds, temperature, etc.  For high-resolution simulations, we recommend using the offline emissions because these emissions are produced at native GEOS-FP and MERRA-2 resolution and then get regridded by HEMCO to your simulation grid.  This would preserve the global totals better.

**Comment by tessac2** [comment]
Thanks @yantosca! I have offline emissions for biogenic, seasalt and soil NOx. I used online for dust emissions, as in version 13.2.1 there was a note in the HEMCO_Config file, "The offline dust emissions are considered to be buggy. Until further notice online dust emissions are recommended."

**Comment by tessac2** [comment]
Hi @yantosca! I had a look at the fractional composition for August, comparing the 4x5 to the 0.25x0.3125. It seems like the OA is much higher in the nested. I used QFED for biomass burning for both, so I am a bit confused why there is such a difference, given qfed is by year/month/day and emissions are already calculated at 1x1 degree resolution, so I would think they would be similar between the two. I was wondering what you recommend for further investigating this issue? I was wondering if comparing the loss rates of wet dep/dry dep between the 4x5 and 0.25x0.3125 at the PBL would make sense? Or if I should look at the budget history collection? Should I try to calculate the lifetime? Please let me know your thoughts/what you recommend! Thank you
![image](https://github.com/geoschem/geos-chem/assets/70339432/3fa1cf29-c9ff-4b8d-86e9-6b43eefb8560)
![image](https://github.com/geoschem/geos-chem/assets/70339432/164793f0-63d0-4e84-95ca-a0eda11c911e)

**Comment by yantosca** [comment]
Hi @tessac2, thanks for the plot.  You could look at the wetdep & drydep loss rates and that might be instructive.  If you used the same biomass burning in both global + nested, one would think that the differences are not caused by the biomass emissions.  

Aerosol chemistry is definitely outside of my wheelhouse, so I would recommend running this by the Aerosols WG.  Tagging @beckyalexander @theloniuspunk @wporter

**Comment by theloniuspunk** [comment]
Hi Tessa, Are you using the simple OA scheme or the complex OA scheme. If the POA is treated a semivolatile, then the dilution of smoke to coarser gridboxes will evaporate some of the OA mass. My guess is that it would not be enough to explain the difference in masses that you have here. 

Otherwise, I agree with Bob's suggestions for next steps.

Jeff

**Comment by wporter** [comment]
Agreed that something looks off with the emissions here -- would be helpful to see HEMCO_Config.rc for both the global and nested simulations.

**Comment by tessac2** [comment]
Thank you @yantosca @wporter @theloniuspunk for your feedback! I have attached the HEMCO_Config.rc (as txt files) for both the global and nested. I am using the simple OA scheme. I will try the budget collection and follow up! 
[HEMCO_Config_GLOBAL.rc.txt](https://github.com/geoschem/geos-chem/files/13605989/HEMCO_Config_GLOBAL.rc.txt)
[HEMCO_Config_NESTED.rc.txt](https://github.com/geoschem/geos-chem/files/13605990/HEMCO_Config_NESTED.rc.txt)

**Comment by yantosca** [comment]
Closing this issue as resolved
